grants_tagger/train.py

```diff
@@ -275,10 +275,22 @@ def train_and_evaluate(
                 pickle.dump(model, f)
         elif y_batch_size: 
             pass # saved already
-        else:
+        else: # dir path that might not exist
             if not os.path.exists(model_path):
                 os.mkdir(model_path)
-            model.save(model_path)
+            if approach in ["cnn", "bilstm"]:
+                # cnn and bilstm are pipelines so do not have a save
+                # but CNNClassifier and BiLSTMClassifier do have one
+                vectorizer = model.steps[0][1]
+                classifier = model.steps[1][1]
+
+                vectorizer_path = os.path.join(model_path, "vectorizer.pkl")
+                with open(vectorizer_path, "wb") as f:
+                    f.write(pickle.dumps(vectorizer))
+                classifier.save(model_path)
```

Comment:
is the classifier save method ok to have a '.pkl' at the end? Usually it's input is just a directory right?